src/abstractdataframe/io.jl

```diff
@@ -307,5 +307,5 @@ DataFrame(sink, sch::Data.Schema, ::Type{S}, append::Bool;
     append!(sink.columns[col], column)
 end
     
-
-Data.close!(df::DataFrameStream) = DataFrame(collect(Any, df.columns), Symbol.(df.header))
+Data.close!(df::DataFrameStream, makeunique::Bool=false) =
```

Comment:
Do we really need this? This implements a particular DataStreams interface which is supposed to be completely generic, so the caller should not have to adapt to `DataFrame` specificities. We should follow the rule adopted by DataStreams in general with regard to duplicate column names.

@quinnj Should we automatically deduplicate column names, or throw an error?

Reply:
I wanted to be explicit rather than implicitly assume one type of behavior without thinking about it. Of course if @quinnj has a clear opinion let us implement it.

Reply:
OK, I've just tested, and it turns out CSV.jl will happily return duplicated column names. So let's always pass `makeunique=true` to `DataFrames`, and remove the keyword argument to `close!`.

Reply:
OK